src/main/java/ch/njol/skript/command/CommandUsage.java

```diff
@@ -46,6 +46,10 @@ public class CommandUsage {
 	 */
 	public CommandUsage(@Nullable VariableString usage, String defaultUsage) {
 		if (usage == null) {
+			// Manually escape quotes. This is not a good solution, as it doesn't handle many other issues, like % in
+			// commands, but in lieu of re-writing the argument parser and command logic completely, I believe this is
+			// a decent stop-gap measure for using " in commands.
+			defaultUsage = defaultUsage.replaceAll("\"", "\"\"");
```

Comment:
Taking another look at this, do you think either of these methods would be useful?
https://github.com/SkriptLang/Skript/blob/0e10975d55b87b742a38ed9f4b01e9cfead1269d/src/main/java/ch/njol/skript/lang/VariableString.java#L270
https://github.com/SkriptLang/Skript/blob/0e10975d55b87b742a38ed9f4b01e9cfead1269d/src/main/java/ch/njol/skript/lang/VariableString.java#L291

Reply:
quote() may work, i'll have to test